stablehlo/dialect/TypeInference.cpp

```diff
@@ -3954,6 +4067,53 @@ LogicalResult verifyReduceWindowOp(
   return success();
 }
 
+LogicalResult verifyReshapeOpQuantizationConstraints(
+    std::optional<Location> location, Type operandTy, Type resultTy) {
```

Comment:
Thanks for suggestion.. I will leave as it is... since won't change wrt readability. usage is contained within the function. So it won't create confusion with other variables.